stock_warehouse_orderpoint_stock_info/models/stock_warehouse_orderpoint.py

```diff
@@ -29,30 +29,60 @@ class StockWarehouseOrderpoint(models.Model):
     product_category = fields.Many2one(
         string="Product Category", related="product_id.categ_id", store=True
     )
+    product_under_minimum = fields.Boolean(
+        string="Product Under Minimum",
+        compute="_compute_product_available_qty",
+        search="_search_product_under_minimum",
+    )
+    product_over_maximum = fields.Boolean(
+        string="Product Over Maximum",
+        compute="_compute_product_available_qty",
+        search="_search_product_over_maximum",
+    )
 
     def _compute_product_available_qty(self):
-        operation_by_locaion = defaultdict(
+        operation_by_location = defaultdict(
             lambda: self.env["stock.warehouse.orderpoint"]
         )
         for order in self:
-            operation_by_locaion[order.location_id] |= order
-        for location_id, order_in_locaion in operation_by_locaion.items():
+            operation_by_location[order.location_id] |= order
+        for location_id, order_in_location in operation_by_location.items():
             products = (
-                order_in_locaion.mapped("product_id")
+                order_in_location.mapped("product_id")
                 .with_context(location=location_id.id)
                 ._compute_quantities_dict(
                     lot_id=self.env.context.get("lot_id"),
                     owner_id=self.env.context.get("owner_id"),
                     package_id=self.env.context.get("package_id"),
                 )
             )
-            for order in order_in_locaion:
+            for order in order_in_location:
                 product = products[order.product_id.id]
                 order.update(
                     {
                         "product_location_qty": product["qty_available"],
                         "incoming_location_qty": product["incoming_qty"],
                         "outgoing_location_qty": product["outgoing_qty"],
                         "virtual_location_qty": product["virtual_available"],
+                        "product_under_minimum": product["qty_available"]
+                        < order.product_min_qty,
+                        "product_over_maximum": product["qty_available"]
+                        > order.product_max_qty,
                     }
                 )
+
+    def _search_product_under_minimum(self, operator, value):
+        if operator == "=":
+            recs = self.search([]).filtered(lambda x: x.product_under_minimum is True)
```

Comment:
@manuelcalerosolis Write a test that searches for product with the field ('product_under_minimum', '=', True).

Moreover, IMHO this is a little bulky as it result of two searches (one with all products, then a search with ids).

Reply:
@rousseldenis certainly the search is very bad, currently the module is not being used in version 13.0 I will prepare it for version 15.0